src/js/VirtualFC.js

```diff
@@ -221,4 +235,22 @@ const VirtualFC = {
     },
 };
 
+const SampleGpsData = {
+    "fix": 2,
+    "numSat": 10,
+    "lat": 474919409,
+    "lon": 190539766,
+    "alt": 0,
+    "speed": 0,
+    "ground_course": 1337,
+    "distanceToHome": 0,
+    "ditectionToHome": 0,
```

Comment:
This is interesting, because I copied the object from memory to clipboard using `copy(FC.GPS_DATA)` in the console. I have to double check this, I think the typo is fundamentally somewhere else...